### DISCUSSION TITLE
How am i supposed to use any of these?

### DISCUSSION THREAD

**Discussion by Yvesaem** [cyberdrop-me/cyberdrop.me-downloaders#6] [Q&A]
pretty much the title sums it up, how do i use these downloaders? do i need to download another program?

**Comment by cyberdrop-me** [comment]
Unless a downloader doesnt have any releases then yes. This windows one [[cy-client]](https://github.com/izqalan/cy-client/releases/tag/v1.1.1) for example has a precompiled release, so you just download it and run it.

Others like the python ones work with python installed, the node ones works with nodejs/npm installed, the shell one works on most linux machines, and so on.
